chapters/methodology/theory.tex

```diff
@@ -1,34 +1,53 @@
-have named the $\Omega$-methods. The foundational research
+have named the $\Omega$-methods. 
+% Maybe \Omega-CADIS-methods? \Omega-methods seems insufficiently specific...?
+The foundational research
 on which the $\Omega$-methods are built will be briefly described to begin this
 section. That discussion will serve as a primer for the subsequent section, which
 is an introduction to the $\Omega$-methods and a discussion on how they differ
 from their predecessors.
 
 \subsection{Previous Work}
+% I this whole section seems to be lit review. I'm not clear about why it is here.
+% I haven't read the lit review section, so maybe the framing makes more sense. 
+% This is why the big outline is helpful...
 \label{sec:omegabknd}
 
-As discussed in sections \ref{sec:CADIS}
+%A thing to note and think carefully about: we probably want to use the words
+%"speed up" about improving calculations. Both acceleration and optimization
+%have specific mathematical meaning; VR methods typically don't do either. I
+%bring this up bc it's something I did not used to be careful about and I got
+%dinged about it by reviewers. If we are careful with wording now then we won't
+%have to worry about anyone being nit-picky later (computational science; all
+%about precision...)
+As discussed in Sections \ref{sec:CADIS}
 through \ref{sec:AngleVR}, the existing gold standard for automatically
 generating variance reduction parameters for deep penetration
 fixed-source radiation transport problems are
 CADIS and FW-CADIS. Both of these methods are very effective at
 generating variance reduction parameters for local and global solutions,
-respectively. However, CADIS and FW-CADIS were developed for optimization in
-space and energy, not angle. As a result, solutions for problems with
-strong anisotropy in the flux are not always optimized with these methods,
-resulting in slow convergence times and low Figure of Merit (FOM) values.
+respectively. However, CADIS and FW-CADIS have only been implemented to
+do variance reduction in space and energy, not angle. 
+% This isn't true. The original documents talk about how it can easily be extended to include angle.
+% It is simply that they have only been implemented for space and energy for a bunch of reasons. 
+As a result, solutions for problems with
+strong anisotropy in the flux are not always improved very efficiently with these methods,
+resulting low Figure of Merit (FOM) values.
 To address this, a number of angle-informed variance reduction
 methods have been
 investigated in the past, most notably AVATAR, LIFT, and a modified version of
 CADIS using AVATAR-type angular parameters.
+% Note: some of these methods were invented -before- CADIS and FW-CADIS. 
+% I think you need to reframe this part as ``we need VR methods that work for problems
+% with strong anisotropies; here are some past attempts.''
+% However, all of that is lit-review discussion... To have it be here you 
```

Comment:
huh. must have had a deletion error (learning mac hot keys, grrr). The point is what we already talked about. Here it can make sense to remind the reader about the details of the pervious methods to illustrate what is different about this method.